mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/impl/MongoDBDatastoreProvider.java

```diff
@@ -136,9 +133,6 @@ protected MongoClient createMongoClient(MongoDBConfiguration config) {
 					? new MongoClient( serverAddresses, clientOptions )
 					: new MongoClient( serverAddresses, credentials, clientOptions );
 		}
-		catch (UnknownHostException e) {
-			throw log.mongoOnUnknownHost( config.toString() );
```

Comment:
It is. I was not sure what policies you have. I can delete the log message in the `Log` class as well, if you like.